docs/program/apis/cloud.md

```diff
@@ -0,0 +1,206 @@
+```
+
+Once you have instantiated an `AppClient`, you can run methods against the `AppClient` object (named `cloud` in the examples) according to the examples for each method below.
+
+## API
+
+The cloud API supports the following methods:
+
+{{< readfile "/static/include/services/apis/cloud.md" >}}
+
+### ListOrganizations
+
+List the organizations the user is an authorized user of.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None
+
+**Returns:**
+
+- [(List[viam.proto.app.Organization])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Organization): A list of the organization or organizations of which the user is an authorized owner.
+
+```python {class="line-numbers linkable-line-numbers"}
+org_list = await cloud.list_organizations()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.list_organizations).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### ListOrganizationMembers
+
+List the members and invites of the organization that you are currently authenticated to.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None
+
+**Returns:**
+
+- [(Tuple[List[viam.proto.app.OrganizationMember]])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.OrganizationMember): A tuple containing to lists:
+  - The first (`[0]`) is a list of organization members.
+  - The second (`[1]`) is a list of organization invites.
+
+```python {class="line-numbers linkable-line-numbers"}
+member_list = await cloud.list_organization_members()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.list_organization_members).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetOrganizationNamespaceAvailability
+
+Check the availability of an organization namespace.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `public_namespace` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The organization namespace to check.
+Namespaces can only contain lowercase alphanumeric and dash characters.
+
+**Raises:**
+
+- GRPCError: An error is raised if an invalid namespace (for example, `""`) is provided.
+
+**Returns:**
+
+- [(bool)](https://docs.python.org/3/library/stdtypes.html#bltin-boolean-values): `True` if the provided namespace is available.
+
+```python {class="line-numbers linkable-line-numbers"}
+available = await cloud.get_organization_namespace_availability(public_namespace="my-cool-organization")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.get_organization_namespace_availability).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### UpdateOrganizationInviteAuthorizations
+
+Update the authorizations attached to an organization invite that has already been created.
```

Comment:
```suggestion
Update (add or remove) the authorizations attached to an organization invite that has already been created.
If an invitation has only one authorization and you want to remove it, delete the invitation instead of using this method.
```